cmd/tnf/generate/catalog/catalog.go

```diff
@@ -54,6 +54,17 @@ var (
 		Short: "Generates the test catalog",
 	}
 
+	generateClassification = &cobra.Command{
+		Use:   "classification",
+		Short: "Generates classification js file",
+	}
+
+	markdownGenerateClassification = &cobra.Command{
```

Comment:
Why not add this command under the catalog node instead of creating a new classification one? The "classification" is more a rendering of the catalog in javascript